### PR TITLE
Fixed colours to use them from global CSS

### PR DESCRIPTION
**Pull request by vaske**
Implemented logic to use colours from global

### CODE REVIEW

**Review by Razzwan — APPROVED** [review]

**Review by orenyodfat — CHANGES_REQUESTED** [review]
there are changes in the colors for some reason .
for example `proposasl page` is purple :
Screen Shot 2020-12-01 at 9 51 06

where before the update it was blue: 
Screen Shot 2020-12-01 at 9 53 04

also when pressing on it it become RED ,..which was not the behaviour before.

**Comment by vaske** [comment]
@orenyodfat I reverted changes and still see same what you had reported, seems like some other changes introduce this

File: src/components/Dao/DaoLandingPage.tsx [hunk 1]
```diff
@@ -28,30 +27,44 @@ export default class DaoLandingPage extends React.Component<IProps, null> {
...
     this.disqusConfig.identifier = this.props.daoState.address;
     this.disqusConfig.title = "Discuss " + this.props.daoState.name;
 
     return (
       <div className={css.landingPage}>
-
-        <BreadcrumbsItem to={"/dao/" + daoState.address}>{daoState.name}</BreadcrumbsItem>
+        <BreadcrumbsItem to={"/dao/" + daoState.address}>
+          {daoState.name}
+        </BreadcrumbsItem>
```
Comment (orenyodfat) [inline src/components/Dao/DaoLandingPage.tsx:67]: why this update ?

**Review by orenyodfat — APPROVED** [review]

**Comment by orenyodfat** [comment]
> @orenyodfat I reverted changes and still see same what you had reported, seems like some other changes introduce this

yes. right . I am seeing that now also in staging.

File: src/components/Dao/DaoLandingPage.tsx (hunk 1 above)
Comment (vaske) [reply src/components/Dao/DaoLandingPage.tsx:67]: my IDE did formatting of JS as I save file

File: src/components/Dao/DaoLandingPage.tsx [hunk 2]
```diff
@@ -63,20 +76,28 @@ export default class DaoLandingPage extends React.Component<IProps, null> {
             </div>
           </div>
 
-          {customData ?
+          {customData ? (
             <>{customData}</>
-            :
+          ) : (
```
[the remark this answers is not in the document]
Comment (vaske) [reply src/components/Dao/DaoLandingPage.tsx:81]: this is just formatting issue

**Pull request merged**